src/Options/index.js

```diff
@@ -347,6 +347,9 @@ export interface ParseServerOptions {
   rateLimit: ?(RateLimitOptions[]);
   /* Options to customize the request context using inversion of control/dependency injection.*/
   requestContextMiddleware: ?(req: any, res: any, next: any) => void;
+  /* If true, disables sanitizing errors and returns the detailed message instead of "Permission denied".
+  :DEFAULT: false */
+  disableSanitizeError: ?boolean;
```

Comment:
```suggestion
  /* If set to `true`, error details are removed from error messages in responses to client requests, and instead a generic error message is sent. Default is `true`.
  :DEFAULT: true */
  enableSanitizedErrorResponse: ?boolean;
```

Note that we may also have log error sanitization in the future, so I've renamed the option.